tests/dhcp_relay/test_dhcp_relay.py

```diff
@@ -178,8 +178,8 @@ def check_routes_to_dhcp_server(duthost, dut_dhcp_relay_data):
 def validate_dut_routes_exist(duthosts, rand_one_dut_hostname, dut_dhcp_relay_data):
     """Fixture to valid a route to each DHCP server exist
     """
-    pytest_assert(check_routes_to_dhcp_server(duthosts[rand_one_dut_hostname], dut_dhcp_relay_data),
-                  "Failed to find route for DHCP server")
+    pytest_assert(wait_until(120, 5, 0, check_routes_to_dhcp_server, duthosts[rand_one_dut_hostname],
```

Comment:
Can you elaborate on why increase the delay? Is there performance degradation?

Reply:
There is a small probability that this case will fail due to route check during testing. I cannot reproduce it manually, so I suspect that the route is not ready. I don’t think there will be too much performance degradation, because in most cases even without this wait_until it can pass, once passed, wait_until will end early.